### ISSUE TITLE
Feedback when change set creation fails due to no changes

### ISSUE THREAD

**Issue by amcinnes** [realestate-com-au/stackup#49]
I want to set up a CI plan that will create a change set, show the user what the changes are, wait for the user to click a button, and then apply the changes.

When updating a stack normally (without change sets), stackup will say "No update required" and exit with code 0 if there are no changes. This makes it easy to implement idempotent deployments.

But when using change-sets, stackup will fail (exit code 1) with:
```
ERROR: The submitted information didn't contain changes. Submit different information to create a change set.
ERROR: change-set creation failed
```

It would be useful if there was some way (exit code) for my script to differentiate between "change set creation failed due to an error" and "change set creation failed due to no changes", so my build can be green when there are no changes needed.

**Comment by mdub** [comment]
It seems like applying an empty change-set should just succeed (exit with code 0), and delete the change-set. Yes?

**Comment by amcinnes** [comment]
Yes, that seems reasonable. But this error happens when _creating_ an empty change-set.

**Comment by mbelekar** [comment]
@amcinnes We can include this change so that build doesn't fail when the change sets is empty.

**Comment by CoreySmithRing** [comment]
> @amcinnes We can include this change so that build doesn't fail when the change sets is empty.

@mbelekar Has this issue been resolved? I have been looking all day for an example or a flag to do exactly this. Change sets with zero changes still goto 'FAILED'. Thank you

**Comment by trembylene** [comment]
Hi @CoreySmithRing 

Unfortunately this issue has not yet been resolved. I have carded this issue up and placed it into our teams backlog. Thank you for letting us know that this is still an issue!

Alternatively, there is also the option for a PR contribution to address this bug in functionality. We welcome PR contributions, and if you tag me or @mukaibot we will see that it is reviewed asap.

**Comment by ps-jay** [comment]
Looks like this issue could be closed with the merger of #90
